### DISCUSSION TITLE
How to solve ReferenceError: structuredClone is not defined

### DISCUSSION THREAD

**Discussion by Velomeloman** [DigiLive/mushroom-strategy#108] [Q&A] [answered]
I have an error:
Error loading the dashboard strategy:

ReferenceError: structuredClone is not defined

Tryed to reinstall strategy, but it still doesn't work.

System log error:
Logger: frontend.js.latest.202312082
Source: components/system_log/__init__.py:300
First occurred: 10:31:11 (1 occurrences)
Last logged: 10:31:11

Uncaught error from Chrome WebView 92.0.4515.105 on Android 12 TypeError: Cannot read property 'showRawConfigEditor' of undefined _showRawConfigEditor (src/panels/lovelace/strategies/device-registry-detail/dialog-dashboard-strategy-editor.ts:181:18) call (src/lit-html.ts:2018:28) dispatchEvent (mwc-list-item-base.ts:247:9) fireRequestSelected (mwc-list-item-base.ts:92:9) call (observer.ts:49:23) forEach (observer.ts:42:28) updated (src/reactive-element.ts:1378:9) _$AE (src/reactive-element.ts:1343:11) performUpdate (src/reactive-element.ts:1261:16)

**Comment by DigiLive** [comment, marked as answer]
structuredClone is unavailable at older JavaScript engines.

Using this method should be replaced in the latest commit.

**Reply by DigiLive** [reply]
@Velomeloman Can you confirm this and mark the comment as an answer if so?
